CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Change Log
 All notable changes to this project will be documented in this file.
 
+## [2.5.0] = 2021-01-29
+-  Adds unfiltered running speed as new data stream
+-  run_demixing gracefully ignores any ROIs that are not in the input trace file
```

Comment:
Sorry. This isn't actually the fix we ended up merging (it broke other downstream modules). All we ended up doing was making sure that exclusion labels added by decrosstalking were "honored" by the demixing module. If a trace is missing from the trace file, demixing will still fail.